dateparser/parser.py

```diff
@@ -142,6 +144,17 @@ def _get_period(cls, format_string):
         else:
             return 'year'
 
+    @classmethod
+    def _find_best_matching_date(cls, datestring):
+        for fmt in cls._preferred_formats_ordered_8_digit:
+            try:
+                dt = strptime(datestring, fmt), cls._get_period(fmt)                
```

Comment:
There are extra spaces at the end here.

Reply:
I have fixed all the whitespace issues in adherence flake8 rules in the latest commit.